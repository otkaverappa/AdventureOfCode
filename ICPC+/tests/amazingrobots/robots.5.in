8 8
########
#......#
#.####.#
..#X...#
#.#....#
#.#....#
#.......
########
2
3 2 3 S
7 6 3 W
8 8
########
#.#....#
#...##.#
..#.#..#
#.#.##.#
...X#..#
#...#...
########
4
3 2 3 E
5 2 2 N
7 7 2 N
6 1 2 E
