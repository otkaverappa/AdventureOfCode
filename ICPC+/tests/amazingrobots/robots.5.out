17
E
N
N
N
N
E
E
E
S
S
S
S
W
S
E
E
E
